src/Service/SearchService.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace App\Service;
+
+use Doctrine\ORM\QueryBuilder;
+
+class SearchService
+{
+    public function getFieldMatches(string $search): array
+    {
+        // Match case number with or without a dash after first two digits
+        preg_match('/^\d{2}-?\d{6}$/', $search, $caseNumberMatches);
+
+        $result = [];
+        if (1 === \count($caseNumberMatches)) {
```

Comment:
This is usually written like

```php
$result = [];

if (preg_match('/^\d{2}-?\d{6}$/', $search, $matches)) {
  // Do stuff with $matches.
  …
}
```

Reply:
Fixed in 79cc63b0254e7d4d3b73c22d6e2fa743eae6e690.